<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButArray">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ArraySequance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButSeq">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ListSequance</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textAllinput">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>411</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>выводl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Butprint">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>82</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>printar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IndexText">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>370</y>
      <width>104</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="GetlsBut">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>330</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Getls</string>
    </property>
   </widget>
   <widget class="QLabel" name="PrintText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>350</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="GetFarBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>GetFirst_ar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetEarBut">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>GetEnd_ar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printlistbut">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>printlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetarBut">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Getar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetFirst_lsBut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GetFirst_ls</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetEndarBut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GetEnd_ar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>968</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
